File: web/src/main/java/no.nav.sbl/rest/domain/RSNyContext.java
package no.nav.sbl.rest.domain;

import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.experimental.Accessors;

@Data
@Accessors(fluent = true)
@EqualsAndHashCode
public class RSNyContext {
    public String verdi;
    public String eventType;
}

File: src/main/java/no/nav/sbl/rest/domain/RSNyContext.kt
package no.nav.sbl.rest.domain;

data class RSNyContext(
    val verdi: String,
    val eventType: String,
    val verdiType: VerdiType? = VerdiType.FNR,
)

enum class VerdiType {
    FNR,
    FNR_KODE
}